digitalpy/core/domain/node.py

```diff
@@ -1,12 +1,12 @@
 import uuid
 import re
 from typing import Dict, Any
-from digitalpy.model.impl.default_persistent_object import DefaultPersistentObject
-from digitalpy.model.load_configuration import Configuration
-from digitalpy.model.object_id import ObjectId
-from digitalpy.model.persistent_object import PersistentObject
-from digitalpy.model.persistent_object_proxy import PersistentObjectProxy
-from digitalpy.model.build_depth import BuildDepth
+from digitalpy.core.persistence.impl.default_persistent_object import DefaultPersistentObject
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position